recipes/darglint/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "darglint" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "d644a3fbe6a80ae73863d79e42be3896447712394930d5e7156bf7476792a243"
+
+build:
+  number: 0
+  entry_points:
+    - darglint = darglint.driver:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

This can be `noarch` and the `pip` arguments are now included by default.

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
  noarch: python
```